### PR TITLE
Start replicaset controller and deployment controller in KCM.

### PR DESCRIPTION
**Pull request by Sindica**

### CODE REVIEW

File: cmd/kube-controller-manager/app/controllermanager.go [hunk 1]
```diff
@@ -392,8 +392,8 @@ func NewControllerInitializers(loopMode ControllerLoopMode) map[string]InitFunc
 	controllers["garbagecollector"] = startGarbageCollectorController
 	controllers["daemonset"] = startDaemonSetController
 	controllers["job"] = startJobController
-	//controllers["deployment"] = startDeploymentController
-	//controllers["replicaset"] = startReplicaSetController
+	controllers["deployment"] = startDeploymentController
+	controllers["replicaset"] = startReplicaSetController
```
Comment (vinaykul) [inline cmd/kube-controller-manager/app/controllermanager.go:396]: Can we also add change to disable workload controller that is launched as part of kube-up?

Comment (Sindica) [reply cmd/kube-controller-manager/app/controllermanager.go:396]: I don't know how to do that in kube-up

**Pull request closed without merging**